src/render/direct3d11/D3D11_PixelShader_HDR10.h

```diff
@@ -0,0 +1,515 @@
+// theTextureUV                      texture  float4          2d             t1      1 
+// Constants                         cbuffer      NA          NA            cb0      1 
+//
+//
+//
+// Input signature:
+//
+// Name                 Index   Mask Register SysValue  Format   Used
+// -------------------- ----- ------ -------- -------- ------- ------
+// SV_POSITION              0   xyzw        0      POS   float       
+// TEXCOORD                 0   xy          1     NONE   float   xy  
+// COLOR                    0   xyzw        2     NONE   float   xyzw
+//
+//
+// Output signature:
+//
+// Name                 Index   Mask Register SysValue  Format   Used
+// -------------------- ----- ------ -------- -------- ------- ------
+// SV_TARGET                0   xyzw        0   TARGET   float   xyzw
+//
+ps_5_0
+dcl_globalFlags refactoringAllowed
+dcl_constantbuffer CB0[5], immediateIndexed
+dcl_sampler s0, mode_default
+dcl_resource_texture2d (float,float,float,float) t0
+dcl_resource_texture2d (float,float,float,float) t1
+dcl_input_ps linear v1.xy
+dcl_input_ps linear v2.xyzw
+dcl_output o0.xyzw
+dcl_temps 4
+sample_indexable(texture2d)(float,float,float,float) r0.x, v1.xyxx, t0.xyzw, s0
+sample_indexable(texture2d)(float,float,float,float) r0.yz, v1.xyxx, t1.zxyw, s0
+add r0.xyz, r0.xyzx, cb0[0].xyzx
+dp3 r1.x, r0.xyzx, cb0[1].xyzx
+dp3 r1.y, r0.xyzx, cb0[2].xyzx
+dp3 r1.z, r0.xyzx, cb0[3].xyzx
+log r0.xyz, |r1.xyzx|
+mul r0.xyz, r0.xyzx, l(0.012683, 0.012683, 0.012683, 0.000000)
+exp r0.xyz, r0.xyzx
+add r1.xyz, r0.xyzx, l(-0.835938, -0.835938, -0.835938, 0.000000)
+max r1.xyz, r1.xyzx, l(0.000000, 0.000000, 0.000000, 0.000000)
+mad r0.xyz, -r0.xyzx, l(18.687500, 18.687500, 18.687500, 0.000000), l(18.851562, 18.851562, 18.851562, 0.000000)
+div r0.xyz, r1.xyzx, r0.xyzx
+log r0.xyz, |r0.xyzx|
+mul r0.xyz, r0.xyzx, l(6.277395, 6.277395, 6.277395, 0.000000)
+exp r0.xyz, r0.xyzx
+mul r0.xyz, r0.xyzx, l(10000.000000, 10000.000000, 10000.000000, 0.000000)
+dp3 r1.x, l(1.660496, -0.587656, -0.072840, 0.000000), r0.xyzx
+dp3 r1.y, l(-0.124547, 1.132895, -0.008348, 0.000000), r0.xyzx
+dp3 r1.z, l(-0.018154, -0.100597, 1.118751, 0.000000), r0.xyzx
+div r0.xyz, r1.xyzx, cb0[4].wwww
+mul r0.xyz, r0.xyzx, cb0[4].zzzz
+mul r1.xyz, r0.xyzx, l(0.012500, 0.012500, 0.012500, 0.000000)
+ne r0.w, l(0.000000, 0.000000, 0.000000, 0.000000), cb0[4].x
+if_z r0.w
+  ge r2.xyz, l(0.250464, 0.250464, 0.250464, 0.000000), r0.xyzx
+  mul r0.xyz, r0.xyzx, l(0.161500, 0.161500, 0.161500, 0.000000)
+  log r3.xyz, |r1.xyzx|
+  mul r3.xyz, r3.xyzx, l(0.416667, 0.416667, 0.416667, 0.000000)
+  exp r3.xyz, r3.xyzx
+  mad r3.xyz, r3.xyzx, l(1.055000, 1.055000, 1.055000, 0.000000), l(-0.055000, -0.055000, -0.055000, 0.000000)
+  movc_sat r1.xyz, r2.xyzx, r0.xyzx, r3.xyzx
+endif 
+mov r1.w, l(1.000000)
+mul o0.xyzw, r1.xyzw, v2.xyzw
+ret 
+// Approximately 36 instruction slots used
+#endif
+
+const BYTE g_main[] =
+{
+     68,  88,  66,  67, 110, 205, 

```

Comment:
Does SDL also commit shader sources?

Reply:
Yep, they’re in this commit.

Reply:
Oops, I didn't see them.
Sorry about the noise.